proto/xudrpc.proto

```diff
@@ -319,21 +319,21 @@ message Chain {
   string network = 2 [json_name = "network"];
 }
 
-message ChannelBalance {
+message Balance {
   // Sum of channel balances denominated in satoshis.
```

Comment:
This should say something like "Total balance denominated in satoshis" instead of "channel balance"